### PR TITLE
bug 1559217: add social-tracker tags

### PR DESCRIPTION
**Pull request by groovecoder**
Adds 5 new lists for social tracking protection.

Note: to test creating these lists, you'll also need to be on [this branch of shavar-list-creation](https://github.com/mozilla-services/shavar-list-creation/pull/83) and add  these corresponding config sections to `shavar_list-creation.ini` (also available in the sample file in the branch):
```
[social-tracking-protection]
disconnect_url=https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/add-socialtracker-tag-1559217/social-tracking-protection-blacklist.json
categories=Facebook,Twitter,LinkedIn,YouTube
output=social-tracking-protection-digest256
s3_key=tracking/social-tracking-protection-digest256

[social-tracking-protection-facebook]
disconnect_url=https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/add-socialtracker-tag-1559217/social-tracking-protection-blacklist.json
categories=Facebook
output=social-tracking-protection-facebook-digest256
s3_key=tracking/social-tracking-protection-facebook-digest256

[social-tracking-protection-twitter]
disconnect_url=https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/add-socialtracker-tag-1559217/social-tracking-protection-blacklist.json
categories=Twitter
output=social-tracking-protection-twitter-digest256
s3_key=tracking/social-tracking-protection-twitter-digest256

[social-tracking-protection-linkedin]
disconnect_url=https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/add-socialtracker-tag-1559217/social-tracking-protection-blacklist.json
categories=LinkedIn
output=social-tracking-protection-linkedin-digest256
s3_key=tracking/social-tracking-protection-linkedin-digest256

[social-tracking-protection-youtube]
disconnect_url=https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/add-socialtracker-tag-1559217/social-tracking-protection-blacklist.json
categories=YouTube
output=social-tracking-protection-youtube-digest256
s3_key=tracking/social-tracking-protection-youtube-digest256
```

### CODE REVIEW

File: social-tracking-protection-blacklist.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{
+ "license": "https://www.mozilla.org/MPL/",
+ "categories": {
+     "Facebook": [
+         {
+             "facebook.com": {
+                 "facebook.com": [
```
Comment (TanviHacks) [inline social-tracking-protection-blacklist.json]: Should we add whatsapp domains?

Comment (englehardt) [reply social-tracking-protection-blacklist.json]: I see that whatsapp.com is already on the bottom of the list. Are you suggesting Luke add other domains from Whatsapp?

**Comment by englehardt** [comment]
I can't comment on the list contents, but the format looks great! It parses correctly in our parser:

```
In [1]: from trackingprotection_tools import DisconnectParser

In [2]: dc = DisconnectParser(blocklist_url='https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/67419dfdf0d18c64c232361a1e1cb9ad036a77ce/social-tracking-protectio
   ...: n-blacklist.json')

In [3]: dc.get_domains_with_category('Facebook')
Out[3]: 
{u'atdmt.com',
 u'atlassolutions.com',
 u'cdninstagram.com',
 u'facebook.com',
 u'facebook.net',
 u'fb.com',
 u'fbcdn.com',
 u'fbcdn.net',
 u'fbsbx.com',
 u'friendfeed.com',
 u'instagram.com',
 u'm.me',
 u'messenger.com',
 u'messengerdevelopers.com',
 u'oculus.com',
 u'oculusbrand.com',
 u'oculusforbusiness.com',
 u'oculusvr.com',
 u'onavo.com',
 u'tfbnw.net',
 u'whatsapp.com'}

In [4]: dc.get_domains_with_category('Twitter')
Out[4]: 
{u'ads-twitter.com',
 u'crashlytics.com',
 u'tweetdeck.com',
 u'twimg.com',
 u'twitter.com'}

In [5]: dc.get_domains_with_category('LinkedIn')
Out[5]: {u'licdn.com', u'linkedin.com'}

In [7]: dc.get_domains_with_category('YouTube')
Out[7]: {u'youtube.com'}
```

**Comment by groovecoder** [comment]
Merging here so we can merge https://github.com/mozilla-services/shavar-list-creation-config/pull/50 to check the lists are created properly for stage.

**Pull request merged**